content/apps/managing-teammates.md

```diff
@@ -0,0 +1,24 @@
+---
+menu:
+  main:
+    parent: advanced
+title: Managing Teammates
+weight: 10
+---
+
+To add a new teammate to your organization:
+
+1. [Send them an invite](https://login.cloud.gov/invitations/new).
+1. Run
+
+    ```bash
+    cf set-org-role <email> <org> OrgManager
```

Comment:
Making them an OrgManager allows them to add other people as well... This should only be done if you want to distribute administrator powers for the org. At 18F maybe that's OK/normal, but I think not in most places. Maybe safer to have this doc suggest "OrgAuditor" (the read-only role) and let them find OrgManager on their own later when they are actually looking for that admin distribution.